### PR TITLE
Fedekunze/1010 lcd stake

### PR DESCRIPTION
**Pull request by fedekunze**
Description:
Updates LCD endpoints for staking.
**DO NOT MERGE** until [#cosmos-sdk/1880](https://github.com/cosmos/cosmos-sdk/pull/1880) has been merged ! 
 

❤️ Thank you!

### CODE REVIEW

**Comment by faboweb** [comment]
We use the prefix "WIP" for our PRs that are not ready to merge. Changed it for you.

File: app/src/renderer/connectors/lcdClient.js [hunk 1]
```diff
@@ -90,12 +90,60 @@ Object.assign(Client.prototype, {
...
+  // getDelegatorValidators: function(delegatorAddr) {
+  //   return req("GET", `/stake/delegators/${delegatorAddr}/validators`).call(this)
+  // },
+  // // Query a validator info that a delegator is bonded to
+  // getDelegatorValidator: function(delegatorAddr, validatorAddr) {
+  //   return req("GET", `/stake/delegators/${delegatorAddr}/validators/${validatorAddr}`).call(this)
+  // },
+
+  // Get a list containing all the validator candidates
+  getValidators: req("GET", "/stakes/validators/"),
```
Comment (faboweb) [inline app/src/renderer/connectors/lcdClient.js]: typo

Comment (fedekunze) [reply app/src/renderer/connectors/lcdClient.js]: for some reason my ESlint wasn't working. I had to delete it and reinstall

**Comment by faboweb** [comment]
blocked on https://github.com/cosmos/cosmos-sdk/issues/1945

**Comment by faboweb** [comment]
Reviewed and approved by @fedekunze

**Pull request merged**